DemoApp/StreamChat/Components/DemoChatChannelListItemView.swift

```diff
@@ -7,7 +7,7 @@ import UIKit
 
 final class DemoChatChannelListItemView: ChatChannelListItemView {
     override var contentBackgroundColor: UIColor {
-        /// In case it is a message search, we don't want change the bg color.
+        // In case it is a message search, we want to ignore the pinning behaviour.
```

Comment:
Although the previous comment has grammar issues, it seems it's meaning was correct?

Reply:
It was, but IMO it was not 100% clear what the goal was. (I wrote the comment myself, and I didn't remember what it was about 😂 )